pjmedia/src/pjmedia-videodev/avi_dev.c

```diff
@@ -671,8 +671,10 @@ static pj_status_t avi_dev_strm_destroy(pjmedia_vid_dev_stream *strm)
 
     avi_dev_strm_stop(strm);
 
-    stream->adi->strm = NULL;
-    stream->adi = NULL;
+    if (stream->adi) {
+        stream->adi->strm = NULL;
+        stream->adi = NULL;
+    }
```

Comment:
Minor: the original codes seem just fine, why adding the check? for special treatments, usually comments can be helpful for future ref.